hardhat.config.ts

```diff
@@ -40,7 +40,13 @@ const config = {
       }
     ],
   },
-  networks: {},
+  networks: {
+    ganashe: {
+      url: 'http://127.0.0.1:7545',
+      accounts: [`0x15da52959efa15bc7a7b3c7bc293425e2e493a626a7c51e77b2b4326adcbe822`],
```

Comment:
that shouldn't be hardcoded (use env instead).

Reply:
fixed